src/domain/enums/DomainErrors.enum.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @enum {DomainErrorMessagesEnum} DomainErrorMessagesEnum - Definition of messages displayed in errors
+ *
+ * @arg {string} AccessDeniedError - Access denied message error
+ * @arg {string} EmailInUseError - Email in use message error
+ * @arg {string} InvalidCredentialsError - Invalid credentials message error
+ * @arg {string} UnexpectedError - Unexpected message error
+ * 
+ */
+export enum DomainErrorMessagesEnum {
+    AccessDeniedError = "Access denied!",
```

Comment:
use 2 spaces and simple quotes in all file

Reply:
I fix this question @rubemfsv